docs/install_notes.rst

```diff
@@ -85,6 +85,20 @@ You can test that this process has worked correctly by going back to the VerifAI
 
 .. _VerifAI: https://github.com/BerkeleyLearnVerify/VerifAI
 
+MacOS
+-----
+
+.. _pythonfcl:
+
+Installing python-fcl on Apple silicon
+++++++++++++++++++++++++++++++++++++++
+If on an Apple-silicon machine you get an error related to pip being unable to install ``python-fcl``, it can be installed manually using the following steps:
+
+1. Clone the `python-fcl <https://github.com/BerkeleyAutomation/python-fcl>`_ repository.
+2. Navigate to the repository.
+3. Install dependencies using homebrew with the following command: ``brew install fcl eigen octomap``
```

Comment:
Commands like this should use the `:command:` role.

Also please link to Homebrew in case people don't have it already.